templates/header.php

```diff
@@ -1,13 +1,11 @@
-<?php $site_icon_url = $this->get( 'site_icon_url' ); ?>
```

Comment:
Sadly, there are more breaking changes than just this one :(

It's why we've including a back-compat mode (#492; automatically enabled for VIPs for now and more details to come) but I guess it makes sense to keep the file name the same here to simplify the upgrade path.